src/app.ts

```diff
@@ -219,12 +231,19 @@ async function runWizard() {
 
   // Ask for source description
 
+  const baseOptions: AICallerOptions = {
+    model: wizardState.smarterModel,
+    maxOutputTokens: 700,
```

Comment:
wai 700?

Reply:
...because that's what you were using?

Reply:
can't argue with myself- although I just pushed a commit contradicting me